### PR TITLE
Feat: Translate guides/release/accessibility/application-considerations.md

### LINKED ISSUE #9: Translate `/accessibility/application-considerations.md`, Ember 4.7
Please assign yourself to the issue or let a comment at the very moment you start the translation.

        File: guides/release/accessibility/application-considerations.md
        From Ember: **N/A**
        To: **4.7**

### PR DESCRIPTION
**Pull request by KamiKillertO**
Closes #9

### CODE REVIEW

File: guides/release/accessibility/application-considerations.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
 
-For existing Ember apps, a developer may edit the `index.html` file or leverage [ember-intl](https://github.com/ember-intl/ember-intl).
+Pour une application existant, un développeur peut éditer le fichier `index.html` file ou installer l'addon [ember-intl](https://github.com/ember-intl/ember-intl).
 
-The `html` element may not have multiple `lang` _values_. If an element contains content in a language different from the primary, then you can provide the element its own `lang` attribute.
+L'attribut `lang` du tag `html` ne peut contenir plusieurs _valeurs_. Si un élément de la page contient du texte dans une langue différente que la langue principale, vous pouvez dans ce cas lui définir son propre attribut `lang`.
 
-![For example, the HTML tag may have a lang of "es" while a paragraph may have a lang of "en"](/images/accessibility/application-considerations/lang.png)
+![Par exemple, the tag HTML peut définir "es" en tant que langue principale et un paragraphe peut quant à lui avoir "en" en tant que langue définie](/images/accessibility/application-considerations/lang.png)
```
Comment (KamiKillertO) [inline guides/release/accessibility/application-considerations.md:22]: @BlueCutOfficial @GreatWizard it looks like assets are missing

File: guides/release/accessibility/application-considerations.md [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
-- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
-- [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
-- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
-  ![Template Linting Preview](/images/accessibility/application-considerations/template-lint.png)
+- [ember-a11y-landmarks](https://github.com/ember-a11y/ember-a11y-landmarks) - Un addon Ember facilitant l'utilisation des rôles sémantique HTML afin d'améliorer l'accessibilité.
+- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - Un mixin pour ajouter des méthodes à vos composants Ember afin de vous aider à gérer l'élément ayant le focus
+- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures fournit un a moyen simple d'utiliser les interactions tactiles en facilitant la définition des contrôler et des reconnaisseurs HammerJS dans une application.
```
Comment (KamiKillertO) [inline guides/release/accessibility/application-considerations.md:49]: If someone has a better translation for recognizers, I'm not a big fan of *reconnaisseurs* 😅

**Review by MrChocolatine — APPROVED** [review]
Non blocking comments, but some of them should be addressed I think.
Approved.

File: guides/release/accessibility/application-considerations.md [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
+Déclarer la langue d'une page HTML permet aux utilisateurs de mieux comprendre son contenu.
 
-> Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. As a result, users with disabilities will be better able to understand the content.
-> [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
+> Les outils d'accessibilités ainsi que les différents navigateurs WEB sont capables d'avoir un rendu plus précis quand la langue d'une page WEB est définie. Les lecteurs d'écrans peuvent ainsi affiner leurs prononciations. Les navigateurs WEB peuvent correctement afficher les caractères et les scripts. Les lecteurs vidéos peuvent afficher les bons sous-titres. Tout ceci permet aux utilisateurs, ayant un handicap, de mieux comprendre le contenu d'une page WEB.
+> [Critère de réussite du WCAG 3.1.1: Intention](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
```
[the remark this answers is not in the document]
Comment (KamiKillertO) [reply guides/release/accessibility/application-considerations.md:8]: It's a good question.
In PR #159 we've stated that we should avoid translating article titles and section titles

File: guides/release/accessibility/application-considerations.md [hunk 4]
```diff
@@ -1,59 +1,59 @@
...
+- [ember-keyboard](https://github.com/patience-tema-baron/ember-keyboard) - Un addon Ember.js pour simplifier la gestion des événements clavier.
+- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - Pour ajouter facilement des tests d'accessibilités dans une suite de tests Ember.
+- [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - Un addon permettant d'annoncer les changements de route côté lecteur d'ecran.
+- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter pour les template Ember
+  ![Aperçu d'un compte rendu de lint d'un template](/images/accessibility/application-considerations/template-lint.png)
 
-While there are quite a few moving parts, here's a cheat sheet to get you started: [Accessibility Cheat Sheet](https://moritzgiessmann.de/accessibility-cheatsheet/)
+Bien qu'il y ait beaucoup de notions à prendre en compte et à connaître, voici une aide-mémoire pour vous aider à commencer: [Aide-memoire Accéssibilités](https://moritzgiessmann.de/accessibility-cheatsheet/)
```
[the remark this answers is not in the document]
Comment (KamiKillertO) [reply guides/release/accessibility/application-considerations.md:59]: Indeed

File: guides/release/accessibility/application-considerations.md (hunk 1 above)
Comment (BlueCutOfficial) [reply guides/release/accessibility/application-considerations.md]: Ooops, ok, we'll have a look 👍

File: guides/release/accessibility/application-considerations.md [hunk 5]
```diff
@@ -1,59 +1,59 @@
...
-- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
-- [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
-- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
-  ![Template Linting Preview](/images/accessibility/application-considerations/template-lint.png)
+- [ember-a11y-landmarks](https://github.com/ember-a11y/ember-a11y-landmarks) - Addon facilitant l'utilisation des rôles sémantique HTML afin d'améliorer l'accessibilité.
+- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - Un mixin pour ajouter des méthodes à vos composants Ember afin de vous aider à gérer l'élément ayant le focus
+- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures fournit un a moyen simple d'utiliser les interactions tactiles en facilitant la définition et l'utilisation de contrôleurs HammerJS dans une application.
```
Comment (BlueCutOfficial) [inline guides/release/accessibility/application-considerations.md:49]:
```suggestion
- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures permet d'intégrer facilement les interactions tactiles en simplifiant la définition et l'utilisation de contrôleurs HammerJS dans une application.
```

File: guides/release/accessibility/application-considerations.md [hunk 6]
```diff
@@ -1,59 +1,59 @@
...
-- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
-- [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
-- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
-  ![Template Linting Preview](/images/accessibility/application-considerations/template-lint.png)
+- [ember-a11y-landmarks](https://github.com/ember-a11y/ember-a11y-landmarks) - Addon facilitant l'utilisation des rôles sémantique HTML afin d'améliorer l'accessibilité.
+- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - Un mixin pour ajouter des méthodes à vos composants Ember afin de vous aider à gérer l'élément ayant le focus
+- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures permet d'intégrer facilement les interactions tactiles en simplifiant la définition et l'utilisation de contrôleurs HammerJS dans une application.
+- [ember-steps](https://github.com/rwjblue/ember-steps) - Addon de création de wizard, de flow avec onglets, et plus ...
```
Comment (BlueCutOfficial) [inline guides/release/accessibility/application-considerations.md:50]: @KamiKillertO we should remove this line. I am currently making a rebase to upstream and I see they removed it. Managing the conflict will be easier if we can just keep your version.

```suggestion
- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - Un mixin pour ajouter des méthodes à vos composants Ember afin de vous aider à gérer l'élément ayant le focus
- [ember-steps](https://github.com/rwjblue/ember-steps) - Addon de création de wizard, de flow avec onglets, et plus ...
```

File: guides/release/accessibility/application-considerations.md (hunk 1 above)
Comment (BlueCutOfficial) [reply guides/release/accessibility/application-considerations.md]: @KamiKillertO you can cherry-pick your work above `master`, now the assets are back in our repo 😃/😭

**Pull request merged**